src/Sylius/Bundle/UserBundle/spec/EventListener/UserLastLoginSubscriberSpec.php

```diff
@@ -17,7 +17,7 @@
 use Sylius\Bundle\UserBundle\Event\UserEvent;
 use Sylius\Bundle\UserBundle\EventListener\UserLastLoginSubscriber;
 use Sylius\Bundle\UserBundle\UserEvents;
-use Sylius\Component\User\Model\UserInterface;
+use Sylius\Component\Core\Model\ShopUser;
```

Comment:
This class is part of User bundle, so we cannot couple it with core. For the basic tests we should use `Sylius\Component\User\Model\User` class. And then, add another test, something like:
```php
     function it_updates_user_last_login_on_security_interactive_login_only_for_class_defined_in_constructor(
          ObjectManager $userManager,
          InteractiveLoginEvent $event,
          TokenInterface $token,
      ) {
          $event->getAuthenticationToken()->willReturn($token);
          $token->getUser()->willReturn(new \stdClass());
 
         $user->setLastLogin(Argument::type(\DateTime::class))->shouldNotBeCalled();
 
         $userManager->persist($user)->shouldNotBeCalled();
         $userManager->flush()->shouldNotBeCalled();
 
         $this->onSecurityInteractiveLogin($event);
     }
```